models/comments.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+exports.selectArticleComments = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments
+    WHERE article_id = $1
+  ;`,
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return { msg: "no comments available" };
```

Comment:
Looking really good! But this might need a bit of refactoring when you do your test for 404 article not found.